`pip3 not found` error in achatina/achatina

When attempting to cross compile `achatina/achatina` on Mac with `ARCH=arm make build`, I get the error below.
```
 => ERROR [6/8] RUN pip3 install requests                                                0.3s
------
 > [6/8] RUN pip3 install requests:
#11 0.272 /bin/sh: pip3: not found
------
executor failed running [/bin/sh -c pip3 install requests]: exit code: 127
make: *** [build] Error 1
```

--- a/achatina/Dockerfile.arm
+++ b/achatina/Dockerfile.arm
@@ -4,6 +4,7 @@ RUN apk update && apk add --no-cache \
   bash \
   mosquitto-clients \
   python3 \
+  py3-pip \
   jq \
   && rm -fr /tmp/*
 

